scripts/validate-cloudtrail-logs.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+##? Usage: validate-cloudtrail-logs.sh [options] ACCOUNT START_FROM [UP_TO]
+##?
+##? Options:
+##?   --config=FILE  Path to file containing details of trails (default: $DM_CREDENTIALS_REPO/jenkins-vars/cloudtrail.yaml)
+##?   -v, --verbose
+##?   -n, --dry-run
+##?   -h, --help
+
+set -eo pipefail
+
+help=$(grep "^##?" "$0" | cut -c 5-)
+eval "$(docopts -h "$help" : "$@")"
+
+VERBOSE="${verbose:-$VERBOSE}"
+$VERBOSE && set -x
+
+[ -n "$config" ] && CONFIG_FILE="$config" || CONFIG_FILE="$DM_CREDENTIALS_REPO/jenkins-vars/cloudtrail.yaml"
+[ -f "$CONFIG_FILE" ] || { echo "error: config file '$CONFIG_FILE' does not exist"; exit 64; }
+
+[ -n "$UP_TO" ] || UP_TO="now"
+START_TIME="$(date --date="$START_FROM" --utc --iso-8601=minutes)" || exit 64
```

Comment:
Don't know what an exit 64 is or why we're doing it. And I don't think it's common knowledge

Reply:
(could probably use a comment)

Reply:
According to docopts it's linux convention https://man.cx/sysexits(3)

I didn't know about it before though so if you didn't either we should probably ignore it, rather than confuse people

Reply:
Removed this